### DISCUSSION TITLE
Paragraph and Spaces

### DISCUSSION THREAD

**Discussion by ConsuelodeIruarrizaga** [twilio-labs/paste#2518] [Q&A] [answered]
Hi everyone, how are you?
I hope everyone is doing fantastic!
I have a question about how we should work with the engineer with the component paragraph: why can't we modify the weight? In some designs, we need to use regular, bold, and semibold. But it [Paste](https://paste.twilio.design/components/paragraph) is saying it isn't modifiable. This is regarding these designs [Figma Link](https://www.figma.com/file/d54HaPcur2mamAwvdx0m1C/2022-Q2-DialogFlow-CX----One-Click-MVP?node-id=141%3A87616)
<img width="736" alt="Screenshot 2022-07-08 at 08 23 02" src="https://user-images.githubusercontent.com/101562667/177933466-8d7b6d64-edb7-48ae-b3af-6065cc7537da.png">

I am going on a PTO from the 11 of July until the 05 of August (both included). Could you please reach out to Kevin Choy for the answer (Engineer from the Voice Automation team). I will also give him the link to this discussion! 

Thank you!!!!
Consuelo

**Comment by nkrantz** [comment, marked as answer]
Hi @ConsuelodeIruarrizaga! The Paragraph component's `font-weight` and other properties are opinionated in order to maintain consistency. We generally encourage folks to be very intentional when applying additional styles to components. If your goal is to add bold styles to words or sentences within long form copy, you should use `<strong>` or `<em>` HTML tags with the Paragraph component. If the bold and semibold are important parts of a UI design like the table in Step 2 of the Figma link you shared, and <strong>not</strong> part of long form copy, we do have a [Text Primitive](https://paste.twilio.design/primitives/text/) that accepts `font-weight` design tokens. You can use that one to make up the table design, just remember to use the Paragraph component for long form text.

Thanks!
